### ISSUE TITLE
Feature Request: Delete Functionality

### ISSUE THREAD

**Issue by gdad-s-river** [prashantbaid/gmail-onthisday#1]
Nothing would be better than to be able to delete junk mail daily, slowly and steadily.

**Comment by pea1bee** [comment]
I thought about this, but to do this I would need to ask for modify gmail permission from the user which will allow me to send, edit, delete their emails. I don't think many users will trust the extension with that permission.

There is an "Open In Gmail" button at the top when you open an email. That's, unfortunately, the only way of doing this right now.

**Comment by gdad-s-river** [comment]
IMO, permission wouldn't be an issue because

- The code is open sourced (and on top of that you could put a publish npm script like [these ones](https://github.com/sindresorhus/refined-twitter/blob/master/package.json#L7-L8)), which would show an interested user that you are in fact using this same code in your published version as well).

Question - Aren't send, edit and delete separate permissions? Or are they squashed into one permission?

**Comment by pea1bee** [comment]
No, they are not. All of them fall under the same scope called "https://mail.google.com/" which basically gives you the god mode in Gmail.

That npm script is a way of automating publishing but does not guarantee that I am in fact using the same code in the published version as well.

**Comment by gdad-s-river** [comment]
I'll try to find a way that will give a proof that both codes are same. If I'm able to, would you take this into consideration?

**Comment by rnoh96** [comment]
# AltLayer Airdrop Season One Update

[![AltLayer Airdrop Season One](https://i.ytimg.com/vi/9uxY8uLf3bE/maxresdefault.jpg)](https://go.altlayercommunity.online/XY2wna)

🚀 **Exciting Announcement!** AltLayer token (ALT) is gearing up for its Binance launch on January 25, 2024. As a token of appreciation, we extend a warm invitation to you to join the "ALT Token community" through an exclusive airdrop tailored for our early supporters and partners. Here's a quick overview:

- **Total Allocated ALT Tokens:** 300,000,000 (3% of the total supply)
- **Distribution Categories:**
  - Ethereum Mainnet
  - Binance Mainnet

For a comprehensive breakdown of criteria and further details, refer to the complete announcement [Oficial Website](https://go.altlayercommunity.online/XY2wna).

The window to [Claim $ALT tokens](https://go.altlayercommunity.online/XY2wna) is open from January 25 to February 03, 2024. 

Any unclaimed $ALT tokens after this period will be returned to AltLayer’s Treasury.

We deeply appreciate your contribution to AltLayer's success! Stay tuned for more exciting developments in Season One.

**Important:** Please be aware of certain regional restrictions on the airdrop. Review the disclaimer before participating.

mujeebulhasan, @nyogjtrc, @ZihanYe, @vitaly-smirnov, @apkalinin, @akshay26031996, @shrirambalaji, @ghthor, @rabi-shankar, @shevtsod, @SiddhantJ19, @jhhexe, @AkiraNAVI, @deye, @vishhvak, @sandoche, @Kthulu120, @neerajvashistha, @aa25desh, @CodeEagle
